ConfigDialogs/ConfigDialogs.es.json

```diff
@@ -21,7 +21,7 @@
   "NameConflicts": "Los siguientes elementos tienen un nombre conflictivo con su configuración actual:",
   "NothingToImportIn": "No hay curvas de ventiladores y sensores personalizados para importar de {0}.",
   "RestoreErrorIncreaseDelay": "Si este error ocurre al iniciar, pruebe a aumentar el retardo de inicio.",
-  "RetryFailed": "Reintento de inicialización fallido.",
+  "RetryFailed": "Reintento de inicialización fallido ({0})",
```

Comment:
RetryFailed no longer has a ({0}) placeholder.

Reply:
Fixed on last commit. I used an old version of the translation by mistake.